doc/development_guide/contribute.rst

```diff
@@ -162,12 +162,14 @@ During development, it's sometimes helpful to run all functional tests in your
 current environment in order to have faster feedback. Run from Pylint root directory with::
 
     python tests/test_functional.py
+    python tests/test_func.py # For legacy functional tests
```

Comment:
I think we should skip these. The intention is to remove them after they are ported to the new functional test framework. This hasn't been documented in a task, but it's something we should do to get rid of them.

Reply:
Good point, I'll remove that from the doc. The error message when a legacy tests fail tells dev what to do anyway.